.github/workflows/lamatic-update.yml

```
@@ -0,0 +1,26 @@
+name: Detect & Sync Lamatic Flows
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)